extensions/applicationinsights-analytics-js/src/JavaScriptSDK/Telemetry/PageViewManager.ts

```diff
@@ -227,9 +231,9 @@ export class PageViewManager {
             };
 
             _self.teardown = (unloadCtx?: IProcessTelemetryUnloadContext, unloadState?: ITelemetryUnloadState) => {
-                if (intervalHandle) {
-                    clearInterval(intervalHandle);
-                    intervalHandle = null;
+                if (queueTimer) {
+                    clearInterval(queueTimer);
```

Comment:
Do we need to change this to clearTimeout here?

Reply:
Oh, nice catch, its actually the other way around this should stay as clearInterval but line 68 should be setInterval

Reply:
Correction, as this is the PageViewManager yes this should be clearTimeout.